### PR TITLE
docs: component-styles guide explains that inline styles must be in CSS

### PR DESCRIPTION
**Pull request by wardbell**
Cannot write them in less, sass, or stylus
See CLI issue https://github.com/angular/angular-cli/issues/8472

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

File: aio/content/guide/component-styles.md [hunk 1]
```diff
@@ -256,10 +257,18 @@ You can write style files in [sass](http://sass-lang.com/), [less](http://lesscs
 ...
 </code-example>
 
-The CLI build process runs the corresponding CSS pre-processors.
+The CLI build process runs the pertinent CSS pre-processor.
 
-You can also configure the CLI to default to your preferred CSS pre-processer 
-as explained in the [CLI documentation](https://github.com/angular/angular-cli/wiki/stories-css-preprocessors).
+When generating a component file with `ng generate component`, the CLI emits an empty CSS styles file (`.css`) by default.
+You can configure the CLI to default to your preferred CSS pre-processer
```
Comment (kapunahelewong) [inline aio/content/guide/component-styles.md]: Typo: pre-processer --> pre-processor

That made me wonder whether it's hyphenated or not. A quick search showed it not hyphenated. Do you feel one way or another about it @wardbell ?

**Review by kapunahelewong — CHANGES_REQUESTED** [review]
LGTM, only a couple small things (in one comment).

File: aio/content/guide/component-styles.md (hunk 1 above)
Comment (wardbell) [reply aio/content/guide/component-styles.md]: Fixed. Ready for you again.

**Review by Foxandxss — APPROVED** [review]

File: aio/content/guide/component-styles.md [hunk 2]
```diff
@@ -179,7 +179,7 @@ to a component's `@Component` decorator:
 <code-tabs>
   <code-pane title="src/app/hero-app.component.ts (CSS in file)" path="component-styles/src/app/hero-app.component.1.ts"></code-pane>
   <code-pane title="src/app/hero-app.component.css" path="component-styles/src/app/hero-app.component.1.css"></code-pane>
-</code-tabs>
+</code-tabs>
```
Comment (petebacondarwin) [inline aio/content/guide/component-styles.md]: trailing whitespace 😱

**Pull request closed without merging**